src/components/number-picker/number-picker.component.spec.ts

```diff
@@ -7,15 +7,15 @@ import { FormBuilder, FormGroup, Validators, ReactiveFormsModule, FormsModule }
     selector: 'app-number-picker-form',
     template: `<ux-number-picker min="-10"
                                  max="10"
-                                 [valid]="form.controls['integer'].valid" 
+                                 [valid]="form.controls['integer'].valid"
                                  [formControl]="form.controls['integer']">
                 </ux-number-picker>
                 <ux-number-picker min="-10"
```

Comment:
If you change this to `[min]="min"` and set up the min property in the component, you can adjust it within the tests by changing `component.min` within the tests. Using this technique, could you test the original issue of modifying the min/max and assuring that the valid state is up to date? The test descriptions will need to be updated as well.

Reply:
My understanding of the component is that by setting `[min]="min"`, all I'm doing is setting the minimum value the user can click down to or wheel down to with the mouse. To change the value that the validation is triggered on, wouldn't I have to change the value that was used is say the formBuilder?

Reply:
If you look at the plunker attached to the ticket (https://plnkr.co/edit/L7P1oNvAi5yDLWJv8rhh?p=preview) you can see that the issue relates to the `max` input, so we do indeed have to test altering `max` (and by extension `min`) to ensure that they update the validation state. Sorry that this was not clear from the ticket description.